test/AllReduceTests.cpp

```diff
@@ -102,6 +102,30 @@ namespace RcclUnitTesting
     testBed.Finalize();
   }
 
+  TEST(AllReduce, Channels)
+  {
+    TestBed testBed;
+    if(testBed.ev.isGfx94) {
+      // Configuration
+      std::vector<ncclFunc_t>     const funcTypes       = {ncclCollAllReduce};
+      std::vector<ncclDataType_t> const dataTypes       = {ncclBfloat16, ncclHalf};
+      std::vector<ncclRedOp_t>    const redOps          = {ncclSum};
+      std::vector<int>            const roots           = {0};
+      std::vector<int>            const numElements     = {2500, 1024};
```

Comment:
1024 will trigger MSCCL, which will be fine.
2500 is too small to guarantee triggering RCCL. Let's use at least 64M elements here.